homeassistant/components/volvooncall.py

```diff
@@ -58,18 +59,20 @@
             {cv.slug: cv.string}),
         vol.Optional(CONF_RESOURCES): vol.All(
             cv.ensure_list, [vol.In(RESOURCES)]),
+        vol.Optional(CONF_REGION): cv.string,
         vol.Optional(CONF_SERVICE_URL): cv.string,
     }),
 }, extra=vol.ALLOW_EXTRA)
 
 
 def setup(hass, config):
     """Set up the Volvo On Call component."""
-    from volvooncall import Connection, DEFAULT_SERVICE_URL
+    from volvooncall import Connection
     connection = Connection(
         config[DOMAIN].get(CONF_USERNAME),
         config[DOMAIN].get(CONF_PASSWORD),
-        config[DOMAIN].get(CONF_SERVICE_URL, DEFAULT_SERVICE_URL))
+        config[DOMAIN].get(CONF_SERVICE_URL),
```

Comment:
**Question:**
I assume the lib handles this internally now?

Reply:
Yes, moved it there, should be backwards compatible ie no breaking change, explicit url can still be specified.